inst/include/Rcpp/Dimension.h

```diff
@@ -39,14 +39,14 @@ namespace Rcpp{
 	            dims = other.dims ;
 	        return *this ;
 	    }
-	    Dimension(const size_t& n1) : dims(1){
+	    Dimension(const int& n1) : dims(1){
```

Comment:
But `int` is signed and `size_t` is not. I think you just reduced the range of numbers which can be expressed here.